tools/tiled_ensemble/ens_config.yaml

```diff
@@ -0,0 +1,50 @@
+pipeline:
```

Comment:
From what I see, `accelerator` is common between `TrainModelJob, PredictJob, SmoothingJob`. `seed` and `normalization_stage` is common between predict `TrainModelJob` and `PredictJob`. While, logically it makes sense to have well defined steps like `post_processing`, since each job is designed to be independent, I have another alternative rather than defining everything under `pipeline`. We can group all parameters that are shared more than once across jobs, and split the others.

```yaml
seed: 42
accelerator: "cpu"
default_root_dir: "results"
tiling:
    ...
normalization_stage:
    ...
threshold:
    ...
data:
    ...

TrainModel:
    metrics:
        ...
    trainer:
        ...

SmoothingJob:
    apply:
    sigma:
    width:

```

We can then pass the common parameters during initialization. Something like this
```python
def _setup_runners(...)->...:
    normalization_stage = args['normalization_stage']
    data  = args['data']
    ParallelRunner(TrainModelJobGenerator(self.root_dir, normalization_stage=normalization_stage, ...))
```

What do you think?

Reply:
That is a great idea! I'll refactor this in this way.